openhim-mapping-mediator/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+
+services:
+  openhim-mapping-mediator:
+    image: jembi/openhim-mediator-mapping
```

Comment:
```suggestion
    image: jembi/openhim-mediator-mapping:v3.0.0
```

Reply:
cool fixed in [695643e](https://github.com/jembi/platform/pull/193/commits/695643ea720c65f3d5bb34d7cf4e430518dbaf71)